src/category_theory/adjunction/reflective.lean

```diff
@@ -102,4 +102,14 @@ begin
   exact mem_ess_image_of_unit_is_iso A,
 end
 
+universes v₃ u₃
+variables {E : Type u₃} [category.{v₃} E]
+
+/-- Composition of reflective functors. -/
+instance reflective.comp (F : C ⥤ D) (G : D ⥤ E) [Fr : reflective F] [Gr : reflective G] :
+  reflective (F ⋙ G) :=
+{ to_is_right_adjoint := ⟨Gr.left ⋙ Fr.left, adjunction.comp _ _ Gr.adj Fr.adj⟩,
```

Comment:
Does it work to just have the proof as `:= {}.`?

Reply:
No

Reply:
Huh I can remove to_is_right_adjoint and to_full, but not to_faithful

Reply:
Is it because its the only one of them that is a Prop maybe?

Reply:
That's weird! I would have expected it to work. `{ to_faithful := by apply_instance, }` does work...